cms/djangoapps/contentstore/views/tests/utils.py

```diff
@@ -41,27 +41,44 @@ def get_preview_html(self, xblock, view_name):
         resp_content = json.loads(resp.content)
         return resp_content['html']
 
-    def validate_preview_html(self, xblock, view_name, can_add=True):
+    def validate_preview_html(self, xblock, view_name, can_add=True, can_reorder=True, can_move=True):
         """
         Verify that the specified xblock's preview has the expected HTML elements.
         """
         html = self.get_preview_html(xblock, view_name)
         self.validate_html_for_add_buttons(html, can_add)
-
-        # Verify drag handles always appear.
-        drag_handle_html = '<span data-tooltip="Drag to reorder" class="drag-handle action"></span>'
-        self.assertIn(drag_handle_html, html)
+        self.validate_html_for_reorder(html, can_reorder)
+        self.validate_html_for_move_button(html, can_move)
```

Comment:
@mushtaqak I think `can_move` will never get called with `False`, can you double check that. If that's the case i'd like to see coverage for this scenario too.